src/lib/viewers/BaseViewer.js

```diff
@@ -979,11 +982,23 @@ class BaseViewer extends EventEmitter {
 
         const options = boxAnnotations.getOptions && boxAnnotations.getOptions();
 
+        // Set disabled ftux cursor flag depending on if the local storage key is set
+        let isDocumentFtuxCursorDisabled = false;
+        let isImageFtuxCursorDisabled = false;
+        if (this.cache.get(DOCUMENT_FTUX_CURSOR_DISABLED_KEY)) {
+            isDocumentFtuxCursorDisabled = true;
+        }
+        if (this.cache.get(IMAGE_FTUX_CURSOR_DISABLED_KEY)) {
+            isImageFtuxCursorDisabled = true;
+        }
+
         const annotatorOptions = this.createAnnotatorOptions({
             annotator: this.annotatorConf,
             features: options && options.features,
             initialMode: this.getInitialAnnotationMode(),
             intl: (options && options.intl) || intlUtil.createAnnotatorIntl(),
+            isDocumentFtuxCursorDisabled,
```

Comment:
We should avoid adding more top-level options, if possible. Should these properties be added and/or excluded from the `features` object?

Reply:
Removed this line as it's no longer needed.